src/aat/resources/features/F-107/S-928.td.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "should create event successfully for an existing case",
+  "_guid_": "S-928",
+  "_extends_": "F-107_Test_Data_Update",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains some OrganisationPolicy fields with incorrect values"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][childContexts][S-927_Create_Case_Data][testData][actualResponse][body][id]}"
+    },
+    "body": {
+      "data": {
+        "OrganisationPolicyField1": {
+          "OrgPolicyCaseAssignedRole": "[Invalid]"
+          },
+        "OrganisationPolicyField2": {
+          "OrgPolicyCaseAssignedRole": "[Invalid]"
+          }
+        },
+      "event_token": "${[scenarioContext][childContexts][S-927-Prerequisite][testData][actualResponse][body][token]}"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_422_Response",
+    "body" : {
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.ValidationException",
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status" : 422,
+      "error" : "Unprocessable Entity",
+      "message" : "[[ANYTHING_PRESENT]]",
+      "path" : "[[ANYTHING_PRESENT]]",
+      "details" : null,
+      "callbackErrors" : null,
+      "callbackWarnings" : null
```

Comment:
I would actually expect an http-400 for this kind of a situation. Could we check with it's developer why this was implemented as 422? Was there anything that had to be kept consistent with? Without a strong rationale to justify, I'd advise an http-400 for this, and there for this scenario would be revealing a defect, in fact.

Reply:
@kiran-yenigala-hmcts has fixed RDM-8993 to return 400 BadRequest for invalid org policy case roles in case creation and update. Tests amended accordingly.